var/spack/repos/builtin/packages/r-truncnorm/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RTruncnorm(RPackage):
+    """Density, probability, quantile and random number generation functions
+    for the truncated normal distribution."""
+
+    homepage = "https://cran.r-project.org/package=truncnorm"
+    url      = "https://cran.rstudio.com/src/contrib/truncnorm_1.0-8.tar.gz"
+    list_url = "https://cran.rstudio.com/src/contrib/Archive/truncnorm"
+
+    version('1.0-8', 'c60cd6555be0dd2ea91e61757439282d')
+
+    depends_on('r-testthat', type=('build', 'run'))
```

Comment:
This is a suggestion, not an actual requirement. You don't need this dependency.

Reply:
Now it is removed.